Components/card/card.css

```diff
@@ -0,0 +1,128 @@
+.vertical-card{
+    max-width: 300px;
+    width: 100%;
+    position: relative;
+    border-radius: 0.3rem;
+    display: flex;
+    flex-direction: column;
+}
+
+.card-image-container{
+    margin-bottom: 1em;
+}
+
+.card-image-container img{
+    border-radius: 0.3rem;
+}
+
+
+.card-text-container{
+    display: flex;
+    flex-direction: column;
+    padding: 0 1rem 0 1rem;
+}
+
+.card-title{
+    position: relative;
+    opacity: 0.90;
+    display: flex;
+    align-items: center;
+    margin-bottom: 0.2em;
+}
+
+
+
+
```

Comment:
Looks good! Just remove empty lines in the code.

Reply:
sure, will do that thanks!